hoprd/rest-api/src/session.rs

```diff
@@ -648,14 +726,8 @@ pub(crate) struct SessionCloseClientRequest {
 /// will be closed.
 #[utoipa::path(
     delete,
-    path = const_format::formatcp!("{BASE_PATH}/session/{{protocol}}"),
-    params(
-            ("protocol" = String, Path, description = "IP transport protocol")
-    ),
-    request_body(
-            content = SessionCloseClientRequest,
-            description = "Closes the listener on the given bound IP address and port.",
-            content_type = "application/json"),
+    path = const_format::formatcp!("{BASE_PATH}/session/{{protocol}}/{{ip}}/{{port}}"),
```

Comment:
Interesting, I would have expected an IP to not conform to the allowed path syntax. But indeed its possible.

Reply:
I'm also wondering if this is going to work via url encoding?